operator/internal/manifests/env_vars.go

```diff
@@ -0,0 +1,111 @@
+package manifests
```

Comment:
Should we rename this file, because it's not "generic environment variables" but only "proxy variables". Maybe `proxy_env.go`?